ouroboros-consensus/src/Ouroboros/Consensus/Storage/VolatileDB/Impl.hs

```diff
@@ -514,7 +515,7 @@ getBlockInfoImpl = getterSTM $ \st hash ->
     ibiBlockInfo <$> Map.lookup hash (currentRevMap st)
 
 getMaxSlotNoImpl ::
-     forall m blockId. IOLike m
+     forall m blockId. (IOLike m, Typeable blockId, StandardHash blockId)
   => VolatileDBEnv m blockId
   -> STM m MaxSlotNo
```

Comment:
Unrelated to this PR, `blockId` should be `blk` (forgotten in a past refactoring), but if we're touching this signature anyway, we should fix it:

```suggestion
     forall m blk. (IOLike m, Typeable blk, StandardHash blk)
  => VolatileDBEnv m blk
  -> STM m MaxSlotNo
```